dts/arm/st/mp1/stm32mp157.dtsi

```diff
@@ -146,6 +146,69 @@
 				label = "GPIOK";
 			};
 		};
+
+		usart2: serial@4000e000 {
```

Comment:
I don't see any u(s)art1, is that intended?

Reply:
USART1 can not be assigned to the Cortex-M4, only Cortex-A7.
https://wiki.st.com/stm32mpu/wiki/USART_internal_peripheral#Peripheral_assignment

Reply:
ok, fine then!